packages/flutterfire_ui/lib/src/i10n/lang/en.dart

```diff
@@ -240,6 +250,16 @@ class EnLocalizations extends FlutterFireUILocalizationLabels {
     this.signInWithEmailLinkSentText =
         "We've sent you an email with a magic link. Check your email and follow the link to sign in",
     this.sendLinkButtonLabel = 'Send magic link',
+    this.credentialAlreadyInUseErrorText =
+        'This credential is already associated with a different user account.',
+    this.verifyEmailTitleText = 'Verify your email',
+    this.emailVerificationInstructionText =
+        'A verification email has been sent to your email address. '
+            'Please check your email and click on the link to verify '
+            'your email address.',
+    this.couldntVerifyEmailText = "We couldn't verify your email address. ",
```

Comment:
There is an extra space here. Is it correct? I deleted it in the Turkish localization file.